holoviews/core/spaces.py

```diff
@@ -708,22 +736,22 @@ def __getitem__(self, key):
         for a previously generated key that is still in the cache
         (for one of the 'open' modes)
         """
-        tuple_key = util.wrap_tuple_streams(key, self.kdims, self.streams)
+        # Split key dimensions and data slices
+        if key is Ellipsis:
+            return self
+        elif key == ():
+            map_slice, data_slice = (), ()
+        else:
+            map_slice, data_slice = self._split_index(key)
```

Comment:
Any chance ``_split_index`` could support the empty tuple directly? What happens with an empty tuple currently? It would be nice if this bit could be:

```python
if key is Ellipsis:
    return self
 map_slice, data_slice = self._split_index(key)
```